doc/classes/ProjectSettings.xml

```diff
@@ -1047,7 +1047,7 @@
 			[b]Note:[/b] Antialiased software skinned polys are not supported, and will be rendered without antialiasing.
 		</member>
 		<member name="rendering/2d/snapping/use_camera_snap" type="bool" setter="" getter="" default="false">
-			If [code]true[/code], forces snapping of 2D viewports to the nearest whole coordinate.
+			[b]Experimental[/b] If [code]true[/code], forces snapping of 2D viewports to the nearest whole coordinate.
```

Comment:
```suggestion
			[b]Experimental:[/b] If [code]true[/code], forces snapping of 2D viewports to the nearest whole coordinate.
```
(Same for the other one.)

But what I would do (additionally to this) is actually put "experimental" in the setting name:
`rendering/2d/snapping/experimental_use_camera_snap` or something.

Reply:
Ok will do. :+1: